client/src/api/request.js

```diff
@@ -0,0 +1,10 @@
+import API from ".";
+
+export const getAllRequests = async () => {
```

Comment:
should add userId in getAllRequest as a parameter, then add userId in headers

Reply:
Done